db/dataset_eval.py

```diff
@@ -13,6 +13,20 @@
 STATUS_DONE = "done"
 STATUS_FAILED = "failed"
 
+EVAL_COLUMNS = ["dataset_eval_jobs.id::text",
+                "dataset_snapshot.dataset_id::text",
+                "dataset_eval_jobs.snapshot_id::text",
+                "dataset_eval_jobs.status",
+                "dataset_eval_jobs.status_msg", 
+                "dataset_eval_jobs.result", 
+                "dataset_eval_jobs.options", 
+                "dataset_eval_jobs.training_snapshot", 
+                "dataset_eval_jobs.testing_snapshot", 
+                "dataset_eval_jobs.created", 
+                "dataset_eval_jobs.updated", 
+                "dataset_eval_jobs.eval_location"]
+ALL_EVAL_COLUMNS = ", ".join([c for c in EVAL_COLUMNS])
```

Comment:
the name of this constant is a bit confusing. We should rename it to something that tells us that how it is different from `EVAL_COLUMNS`, maybe `eval_columns_comma_seperated`? 

Another thing that we could do is just keep the list and create the comma seperated string inside the query functions.